meta-fobnail-distro/recipes-core/systemd/systemd-conf_%.bbappend

```diff
@@ -0,0 +1,3 @@
+do_install:append(){
```

Comment:
@kotylamichal why we remove those?

Reply:
`systemd` creates configs that override our fobnail token configuration. I will add an appropriate comment to this append

Reply:
@kotylamichal we only add network config for usb interface, which config overrides this?

Reply:
Also, it is recommended to prefix config files with number - more here https://www.freedesktop.org/software/systemd/man/systemd.network.html#Description